js/curation-tool.js

```diff
@@ -1091,5 +1154,66 @@ const vm = new Vue({
 
       return nodeLabelsWithPrefix;
     },
+
+    // Reasoning over phyloreferences
+    reasonOverPhyloreferences() {
+      // Reason over all the phyloreferences and store the results on
+      // the Vue model at vm.reasoningResults so we can access them.
+
+      $('.reason-over-phylorefs').html('(reasoning)');
+      $('.reason-over-phylorefs').prop('disabled', true);
+      $.post(JPHYLOREF_REASON_URL, {
+        jsonld: JSON.stringify([new PHYXWrapper(this.testcase).asJSONLD()], undefined, 4),
```

Comment:
Currently, the POST request is expected to be sent in either `application/x-www-form-urlencoded` or `multipart/form-data`, with either a `jsonld` field containing a JSON-LD string to be reasoned over (as in this code) or a `jsonldFile` field containing a file upload of a JSON-LD file. That way, we allow for additional options to be passed to JPhyloRef if needed. I've added a comment to document this a bit better. Supporting form submissions makes it pretty easy to use `curl` to send requests to JPhyloRef.

If it would be useful to additionally accept POST data directly as `application/json`, let me know and I'll open an issue for that!